sdk/confidentialledger/azure-confidentialledger/azure/confidentialledger/_operations/_patch.py

```diff
@@ -184,23 +189,24 @@ def operation() -> JSON:
 
     def begin_create_ledger_entry(
         self,
-        entry: Union[JSON, IO],
+        entry: Union[_models.LedgerEntry, JSON, IO[bytes]],
         *,
         collection_id: Optional[str] = None,
         **kwargs: Any,
-    ) -> LROPoller[JSON]:
+    ) -> LROPoller[_models.TransactionStatus]:
```

Comment:
This code need some attention, the true return type here should be LROPoller[LedgerEntry] however due to the underlying usage of the begin_wait_for_commit() method the poller seems to return transactionstatus instead